lib/main_containers.pm

```diff
@@ -131,15 +136,21 @@ sub load_host_tests_containerd_nerdctl {
     loadtest 'containers/containerd_nerdctl';
 }
 
-sub load_host_tests_helm() {
+sub load_host_tests_helm {
+    my ($run_args) = @_;
+    my $backends = undef;
+
     if (is_sle('15-sp3+')) {
-        loadtest "containers/helm_eks";
-        loadtest "containers/helm_k3s";
+        $backends = get_var("HELM_K8S_BACKEND","EKS,AKS,K3S");
     } elsif (is_opensuse) {
-        loadtest "containers/helm_k3s";
+        $backends = get_var("HELM_K8S_BACKEND","K3S");
+    } else {
+        die("Helm backend not supported on this host");
     }
-    else {
-        die("Helm test not supported on this host");
+
+    foreach (split(',\s*', $backends)) {
+        push @{$run_args->{backends}}, $_;
```

Comment:
So, for my own understanding, you are creating a new array `@{$run_args->{backends}}` and then pushing the backends in this array? So, for SLE case the array would be
`@{$run_args->{backends}} = ('EKS' ,'AKS', 'K3S')` ?

But how is this handled by the test later on? 
`my $k8s_backend = shift(@{$run_args->{backends}});`